packages/hypothesis/meta.yaml

```diff
@@ -0,0 +1,17 @@
+package:
+  name: hypothesis
+  version: 6.79.2
+  tag:
+    - core
+  top-level:
+    - _hypothesis_ftz_detector
+    - _hypothesis_pytestplugin
```

Comment:
I know that these are auto-generated from mkpkg, but I think we can remove these from top-level since users will not import these modules directly.